PyRDF/backend/Local.py

```diff
@@ -2,19 +2,29 @@
 from .Backend import Backend
 
 class Local(Backend):
+    """
```

Comment:
"""
Backend that relies on the C++ implementation of RDataFrame to locally execute the current graph.
"""